### PR TITLE
parallelized lipidblast and reads more info

### PR DESCRIPTION
**Pull request by pratarora**
I changed compound_tbl_lipidblast and dependent function to allow it to read more information from the json files,and parallelized it to allow faster reading.

I have tested it on old and new lipidblast as described in https://github.com/rformassspectrometry/CompoundDb/issues/114
Best regards
Prateek

### CODE REVIEW

**Review by jorainer — CHANGES_REQUESTED** [review]
Thanks for the contribution @pratarora ! Code looks great, I will then fix the unit tests after merging. 

Can you however please add yourself as a contributor (`role = c("ctb")`) to the DESCRIPTION file and to the `@author tag of the `compound_tbl_lipidblast()` function?

File: R/createCompDbPackage.R [hunk 1]
```diff
@@ -132,16 +132,26 @@ compound_tbl_sdf <- function(file, collapse, onlyValid = TRUE,
...
+#' - `smiles`: the SMILES representation of the compound.
 #' - `formula`: the chemical formula.
 #' - `exactmass`: the compound's mass.
+#' - `compound_class`: the class of the compound.
+#' - `ionization_mode`: the ionization mode.
+#' - `precursor_mz`: the precursor m/z value.
+#' - `precursor_type`: the precursor type.
+#' - `retention_time`: the retention time.
+#' - `ccs`: the collision cross-section.
+#' - `spectrum`: the spectrum data.
```
[the remark this answers is not in the document]
Comment (pratarora) [reply R/createCompDbPackage.R:154]: Thanks @jorainer ! done with the changes.

**Review by jorainer — APPROVED** [review]
Thanks @pratarora ! Will merge and then push to Bioconductor

**Comment by jorainer** [comment]
@pratarora , hope it's OK, but I added you also as a contributor to the [RforMassSpectrometry page](https://www.rformassspectrometry.org/contributions/)

**Comment by pratarora** [comment]
Thanks a lot, @jorainer !

**Pull request merged**